lib/fuse.c

```diff
@@ -4702,6 +4705,12 @@ struct fuse *fuse_new_31(struct fuse_args *args,
 		}
 	}
 
+
+	if (f->conf.max_idle_threads == -1) {
+		/* If the number of idle threads was not set, default to 10. */
+		f->conf.max_idle_threads = 10;
+	}
+
```

Comment:
Why set it to -1, only to later set it to 10? Can't you just initialize it with the default value of 10 (and avoid the need to support negative values)?